### PR TITLE
added the unlgoed use path of interaction

### PR DESCRIPTION
**Pull request by Eric-Durr**

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -2,3 +2,4 @@
 
 * [Introduction](README.md)
 * [Front_End](./front-docs/LANDING_PAGE.md)
+* [unloged use](./front-docs/UNLOGED_USE.md)
```
Comment (delirial) [inline SUMMARY.md]: We need coherency with format

**Review by delirial — DISMISSED** [review]
Good job, when you can have time please upload images for more clarity

File: SUMMARY.md (hunk 1 above)
Comment (Eric-Durr) [reply SUMMARY.md]: ???

**Review by delirial — APPROVED** [review]
Good job!

**Pull request merged**